packages/dashboard-message-bus/lib/utils.ts

```diff
@@ -0,0 +1,158 @@
+import WebSocket, { ServerOptions } from "isomorphic-ws";
+import type { Message, PortsConfig } from "./types";
+import any from "promise.any";
+import delay from "delay";
+import axios from "axios";
+
+any.shim();
+
+/**
+ * Convert any JS object or value to a base64 representation of it
+ */
+export const jsonToBase64 = (json: any) => {
+  const stringifiedJson = JSON.stringify(json);
+  const buffer = Buffer.from(stringifiedJson);
+  const base64 = buffer.toString("base64");
+
+  return base64;
+};
+
+/**
+ * Convert the base64 representation of a JS object or value to its JS representation
+ * @dev This is the reverse of `jsonToBase64` and is not expected to work with other base64 formats
+ */
+export const base64ToJson = (base64: string) => {
+  const buffer = Buffer.from(base64, "base64");
+  const stringifiedJson = buffer.toString("utf8");
+  const json = JSON.parse(stringifiedJson);
+
+  return json;
+};
```

Comment:
If we do `jsonToBase64` and `base64ToJson` elsewhere in truffle, it might make sense to push them up into a higher level truffle util. From a quick cursory glance, I found [one here](https://github.com/trufflesuite/truffle/blob/f83191a654d29750c1c3d84e354085a3b33b646c/packages/environment/develop.js#L34) but if that's it, we could also wait for a rule of 3. (not blocking)